presentation/src/main/java/com/knocklock/presentation/setting/password/PasswordInputViewModel.kt

```diff
@@ -53,12 +53,12 @@ class PasswordInputViewModel @Inject constructor(
         }
 
         if (state.inputPassword.length + 1 >= MAX_PASSWORD_LENGTH) {
```

Comment:
```suggestion
        if (passwordInputState.inputPassword.length >= MAX_PASSWORD_LENGTH) {
```

싱크 맞추려면 이 변수도 `passwordInputState`로 해주면 더 좋겠네요 :0

Reply:
@jaeryo2357 [ace5662](https://github.com/Knock-Lock/Knock-Lock/pull/86/commits/ace566204805dbe9552e721a34418cac977e0c10) 반영했어영